unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ChatController/ChatControllerContextMenuUtility.cs

```diff
@@ -0,0 +1,177 @@
+        }
+
+        [ContextMenu("Fake Private Message")]
+        public void FakePrivateMessage()
+        {
+            UserProfile ownProfile = UserProfile.GetOwnUserProfile();
+
+            var model = new UserProfileModel()
+            {
+                userId = "test user 1",
+                name = "test user 1",
+            };
+
+            UserProfileController.i.AddUserProfileToCatalog(model);
+
+            var model2 = new UserProfileModel()
+            {
+                userId = "test user 2",
+                name = "test user 2",
+            };
+
+            UserProfileController.i.AddUserProfileToCatalog(model2);
+
+            var msg = new ChatMessage()
+            {
+                body = "test message",
+                sender = model.userId,
+                recipient = ownProfile.userId,
+                messageType = ChatMessage.Type.PRIVATE,
+                timestamp = (ulong) DateTimeOffset.UtcNow.ToUnixTimeMilliseconds()
+            };
+
+            var msg2 = new ChatMessage()
+            {
+                body = "test message 2",
+                recipient = model2.userId,
+                sender = ownProfile.userId,
+                messageType = ChatMessage.Type.PRIVATE,
+                timestamp = (ulong) DateTimeOffset.UtcNow.ToUnixTimeMilliseconds()
+            };
+
+            AddMessageToChatWindow(JsonUtility.ToJson(msg));
+            AddMessageToChatWindow(JsonUtility.ToJson(msg2));
+        }
+
+        [ContextMenu("Add fake messages to nearby")]
+        public void AddManyFakeMessagesToNearby()
+        {
+            var users = new List<UserProfileModel>();
+            
+            for (var i = 0; i < 10; i++)
+            {
+                var userId = Guid.NewGuid().ToString();
+                var userProfile = new UserProfileModel
+                {
+                    userId = userId,
+                    name = $"TestUser{i}"
+                };
+
+                UserProfileController.i.AddUserProfileToCatalog(userProfile);
+                users.Add(userProfile);
+            }
+
+            for (var i = 0; i < 100; i++)
+            {
+                var user = users[Random.Range(0, users.Count)];
+                
+                var msg = new ChatMessage
+                {
+                    body = "test message",
+                    sender = user.userId,
+                    recipient = null,
+                    messageType = ChatMessage.Type.PUBLIC,
+                    timestamp = (ulong) DateTimeOffset.UtcNow.ToUnixTimeMilliseconds(),
+                    isChannelMessage = true,
+                    messageId = Guid.NewGuid().ToString(),
+                    senderName = user.name
+                };
+                
+                AddMessageToChatWindow(JsonUtility.ToJson(msg));
+            }
+        }
+        
+        [ContextMenu("Add fake messages to global")]
+        public void AddManyFakeMessagesToUnityChannel()
```

Comment:
If we're going to leave these fake methods to test, I think we should make them to call each `AddMessageToChatWindow` in a different frame to simulate a real case.